test/unit/query-generator/add-column-query.test.ts

```diff
@@ -0,0 +1,35 @@
+import { DataTypes } from '@sequelize/core';
+import { buildInvalidOptionReceivedError } from '@sequelize/core/_non-semver-use-at-your-own-risk_/utils/check.js';
+import { expectsql, getTestDialect, sequelize } from '../../support';
+
+const dialectName = getTestDialect();
+
+describe('QueryGenerator#addColumnQuery', () => {
+  const queryGenerator = sequelize.getQueryInterface().queryGenerator;
+
+  const User = sequelize.define('User', {
+    firstName: DataTypes.STRING,
+  }, { timestamps: false });
+
+  it('generates a ADD COLUMN query in supported dialects', async () => {
+    const sql = queryGenerator.addColumnQuery(User.tableName, 'age', DataTypes.INTEGER);
+
+    expectsql(sql, {
+      default: `ALTER TABLE "public"."Users" ADD COLUMN "age" INTEGER;`,
+      postgres: `ALTER TABLE "public"."Users" ADD COLUMN "age" INTEGER;`,
+      mysql: 'ALTER TABLE `Users` ADD `age` INTEGER;',
```

Comment:
The MariaDB implementation does not override `addColumnQuery`. Haven't looked into whether it should or not